lib/logutils/logutils.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016-2019 Tigera, Inc. All rights reserved.
+// Copyright (c) 2016-2019,2021 Tigera, Inc. All rights reserved.
```

Comment:
I would simplify that to `2016-2021`.  Not saying that you necessarily should too, but curious to know if what I would do would be wrong.

Reply:
I _think_ you're only supposed to include the years when changes were made but I'm not sure it truly matters when we've got git history to precisely track every change.